docs: add a section to the quickstart document that describes how to use the values.yaml file (#42428)
airbytehq/airbyte: docs/using-airbyte/getting-started/oss-quickstart.md

3: [Set up a Connection](./set-up-a-connection.md)

## Customizing your Installation with a Values file

> The `abctl` binary uses Helm to install Airbyte. The `abctl local install` command accepts a values.yaml as an optional 

Reviewer: ```suggestion
Optionally, you can use a `values.yaml` file to customize your installation of Airbyte. Create the `values.yaml` on your local storage. Then, apply the values you've defined by running the following command and adjusting the path to the `values.yaml` file as needed:
```

> parameter. The values.yaml is used to customize your installation of Airbyte. To use a values.yaml file, you must create

Reviewer: ```suggestion

```

> the values.yaml somewhere on your local storage. You can apply the values.yaml by running the following command, 

Reviewer: ```suggestion

```

> adjusting the path to the values.yaml file as needed:

Reviewer: ```suggestion

```
